ci-operator/config/ansible/tower-qa/ansible-tower-qa-devl__ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aap-interop-aws
+  cluster_claim:
```

Comment:
Why are we using the stackrox teams cluster-claim here for a cluster instead of using our AWS account?